plugins/org/src/components/Cards/OwnershipCard/OwnershipCard.tsx

```diff
@@ -88,16 +94,20 @@ export const OwnershipCard = (props: {
   relationsType?: EntityRelationAggregation;
   relationAggregation?: EntityRelationAggregation;
   entityLimit?: number;
+  scrollHeight?: string;
 }) => {
   const {
     variant,
     entityFilterKind,
     hideRelationsToggle,
-    entityLimit = 6,
+    entityLimit,
```

Comment:
Changing the default entity limit is a visual breaking change. We'd be better off finding a solution that doesn't change this.